core/src/main/java/com/kin/ecosystem/core/data/order/OrderRemoteData.java

```diff
@@ -243,6 +244,50 @@ public void getFilteredOrderHistory(@Nullable String origin, @NonNull String off
         getHistory(origin, offerID, ONE_ORDER_LIMIT, callback);
     }
 
+    @Override
+    public void changeOrder(@NonNull final String orderID, @NonNull Body body, @NonNull final Callback<Order, ApiException> callback) {
+        try {
+            ordersApi.changeOrderAsync(orderID, body, new ApiCallback<Order>() {
+				@Override
+				public void onFailure(final ApiException e, int statusCode, Map<String, List<String>> responseHeaders) {
+                    executorsUtil.mainThread().execute(new Runnable() {
+                        @Override
+                        public void run() {
+                            callback.onFailure(e);
+                        }
+                    });
+				}
+
+				@Override
+				public void onSuccess(final Order result, int statusCode, Map<String, List<String>> responseHeaders) {
+                    executorsUtil.mainThread().execute(new Runnable() {
+                        @Override
+                        public void run() {
+                            callback.onResponse(result);
+                        }
+                    });
+				}
+
+				@Override
+				public void onUploadProgress(long bytesWritten, long contentLength, boolean done) {
```

Comment:
why are those callbacks needed? what do we upload here? I understand it's auto generated from the swagger definition, but still wondering

Reply:
no need, generated swagger